### DISCUSSION TITLE
how to change default deliver port from 25 to 465

### DISCUSSION THREAD

**Discussion by ermazi** [emersion/maddy#458] [Q&A]
log:
queue: delivery attempt failed  {"domain":"troy.wang","io_op":"dial","msg_id":"72dca686","rcpt":"me@xxx.xxx","reason":"dial tcp 157.148.57.77:25: connect: connection timed out","remote_addr":"157.148.57.77:25","smtp_code":451,"smtp_enchcode":"5.4.0","smtp_msg":"No usable MXs, last err: dial tcp 157.148.57.77:25: connect: connection timed out","target":"remote"}
cloud provider ban 25 port outbounding, how to change 25 to 465?

**Comment by foxcpp** [comment]
There is no way to do so. Delivery via 465 won't work with most providers. You should either get your provider to unblock port 25 or forward messages via an external host that does not have it blocked.

**Reply by ermazi** [reply]
any example?

**Reply by foxcpp** [reply]
For forwarding via an external host, change
```
target.remote outbound_delivery {
  ...
}
```
to 
```
target.smtp outbound_delivery  {
  targets tcp://relay.example.org:2525
  require_tls 
  auth plain USERNAME PASSWORD
  # any additional config per https://github.com/foxcpp/maddy/blob/dev/docs/reference/targets/smtp.md
}
```

**Reply by ermazi** [reply]
ty, ill try it.

**Reply by ermazi** [reply]
em, i use setup another maddy as smtp relay but failed, any special conf for relay smtp that only forwards email?
